@ecomplus/storefront-app/src/components/js/EcOrderInfo.js

```diff
@@ -218,6 +225,27 @@ export default {
       }
     },
 
+    buyAgain () {
+      const { localOrder } = this
+      if (localOrder.items) {
+        const { items } = localOrder
+        let countAddItems = 0
+        items.forEach(item => {
+          ecomCart.addItem(item)
+          countAddItems++
+          if (countAddItems === localOrder.items.length) {
+            this.$toast({
+              title: `${this.i19updatedCart}`,
+              body: `${this.i19addToCartMsg}`,
```

Comment:
```suggestion
              title: this.i19updatedCart,
              body: this.i19addToCartMsg,
```

Não há concatenação de string, não precisamos das aspas aqui...